packages/styles/scss/components/structured-list/_structured-list.scss

```
@@ -0,0 +1,226 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@use '../../config' as *;
+@use "../../type" as *;
+@use '../../motion';
+@use '../../theme' as *;
+@use "../../utilities/focus-outline" as *;
+@use "../../utilities/skeleton" as *;
+@use "../../utilities/high-contrast-mode" as *;
+
```

```suggestion
@use '../../utilities/convert' as *;
```

Since there is a reference to the `rem` function on L137, we'll need to bring in the convert functions. Or we can change this line to `2.5rem` since it's the only instance in this file